e2e/features/token.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Using mTokens
+  As a user
+  I want to be able to transfer and mint mTokens
+
+  Background:
+    Given I have the following accounts:
+      | account | password | tokens | funds | validator |
+      | A       | test     | 20     | 10    | true      |
+      | B       | test     | 10     | 10    | false     |
+      | C       | test     | 0      | 10    | false     |
+
+  Scenario: Transfer tokens: not-empty receiving account
+    Given I start validator with 5 mTokens deposit
+    And I wait for my node to be synced
+    And the deposit of A should be 15 mTokens
+    And I unlock the account A with password 'test'
+    When I transfer 10 mTokens from A to B
+    Then the deposit of A should be 5 mTokens
+    And the deposit of B should be 20 mTokens
+
+  Scenario: Transfer tokens: empty receiving account
+    Given I start validator with 5 mTokens deposit
+    And I wait for my node to be synced
+    And the deposit of A should be 15 mTokens
+    And I unlock the account A with password 'test'
+    When I transfer 10 mTokens from A to C
+    Then the deposit of A should be 5 mTokens
+    And the deposit of B should be 10 mTokens
```

Comment:
This is working by luck. You should be testing with `C`, and it's passing because B also has 10. Probably changing the number would be a good idea to avoid this kind of false positives.